hapi/hapi.d.ts

```diff
@@ -1873,6 +1873,7 @@ declare module "hapi" {
 			}
 			});*/
 			strategy(name: string, scheme: any, mode?: boolean | string, options?: any): void;
+			strategy(name: string, scheme: any, modeOrOptions: any): void;
```

Comment:
It's not clear to me exactly what you're pointing at, but if it's about treating `options` as optional, I had considered that but that case has already been covered in the first method which has the last two args as optional.

If it's about the name of `options` itself, I guess I'll agree though I was just trying to make it more explicit.

However, I'm surprised I missed `scheme: any` myself. It should be `scheme: string`, which I'll fix in the next commit post this review.